registration/register_node_led.go

```diff
@@ -121,7 +122,60 @@ func FetchNodeCredentials(
 	reqInfo, err := validateFetchRequestCommon(ctx, storage, req, opt...)
 	switch {
 	case err != nil:
-		return nil, fmt.Errorf("(%s) error during fetch request validation: %w", op, err)
+		err := fmt.Errorf("(%s) error during fetch request validation: %w", op, err)
+		return nil, err
+
+	case len(reqInfo.WrappedRegistrationInfo) > 0 || len(req.RewrappedWrappingRegistrationFlowInfo) > 0:
+		var registrationInfo *types.WrappingRegistrationFlowInfo
+		if len(req.RewrappedWrappingRegistrationFlowInfo) > 0 {
+			encryptingNodeInfo, err := types.LoadNodeInformation(ctx, storage, req.RewrappingKeyId, opt...)
+			if err != nil {
+				err := fmt.Errorf("(%s) error looking up node information for rewrapping key id: %w", op, err)
+				opts.WithLogger.Error(err.Error())
```

Comment:
typically not a fan of logging in this sort of module... but this is def needed
in this one.

Reply:
Keep in mind that the idea is that it's not generally logging unless you actually pass it a logger. And that could be one that's only turned on at certain times.

However there is basically no other way to understand what's going on when things aren't working. In those situations activating the logging is invaluable.